cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
+  var _ = function ( elem, opts ) {
```

Comment:
Can this get a more descriptive name? `_` seems confusing as it's the convention of underscore.js.